OpenDreamShared/Compiler/DM/DMParser.cs

```diff
@@ -1343,6 +1343,25 @@ public partial class DMParser : Parser<Token> {
             return expression;
         }
 
+        public DMASTExpression ExpressionEquivalence()
```

Comment:
Equivalence should have the same operator precedence as `==` and `!=`. You can do this by instead adding them to the `ExpressionComparison()` method.

Reply:
Fixed